feature-libs/quote/components/summary/actions/quote-summary-actions.component.html

```diff
@@ -2,6 +2,17 @@
   <ng-container *ngIf="cartDetails$ | async as cartDetails">
     <ng-container *ngIf="areButtonsRendered(quoteDetails.allowedActions)">
       <section [ngClass]="isFixedPosition ? 'fixed' : 'sticky'">
+        <button
+          type="button"
+          id="downloadBtn"
+          class="btn btn-primary"
```

Comment:
I think it's not ok to have 2 primary buttons. The new button should be secondary, and it should appear below the standard action buttons. But I will also let @Larisa-Staroverova , our UX advocate, review the PR

Reply:
If one adds a bunch of items to the quote, then the order summary behaves as expected, namely scrolls up and down: 
![image](https://github.com/SAP/spartacus/assets/61147963/9f7bf2c8-2fbd-42d1-ab86-455437c605c0)
On mobile widgets as well: 
![image](https://github.com/SAP/spartacus/assets/61147963/89245436-3f8e-460d-9862-896f48cbabfe)

Reply:
@suprishi, could you share who your UI designer is?

Reply:
There is no UI designer, I referenced the Accelerator UI and used my best judgement to place the button.

Reply:
Zhanara Baisalova is the UI designer and responsible for a quote library.
I sent her your user story but it is really short and not clear what the use case for downloading proposal is?
What are they actually downloading? What kind of info? 
Please, extend the user story with the corresponding information.
Thanks a lot in advance!

Reply:
Could you attach the corresponding Accelerator UI screenshot?

Reply:
Here's the screenshot from Accelerator
![image](https://github.com/SAP/spartacus/assets/28841459/8ea520d8-f3fd-47f1-89c4-934b8301e373)